uiowa/uiowa01
# Feature: Collegiate theme breakpoints/containers

Work on cleaning up breakpoints and containers for #268.

# How to Test

1. Run `drush @theming.local si collegiate`
2. Create a Basic Page of content
3. Resize page to see if proper padding is being added when content hits side of browser 
4. Edit Layout and add a full width section
5. Change theme settings to full width
6. Edit Layout and adjust section to fixed width
7. Save Layout and test for padding at breakpoint
